Content.Shared/Administration/AdminFlags.cs

```diff
@@ -104,6 +104,11 @@ public enum AdminFlags : uint
 		///</summary>
 		Adminchat = 1 << 17,
 
+        ///<summary>
+        /// Allows you to use Admin chat
```

Comment:
Needs a description that isn't copy pasted.

Reply:
💀

Reply:
done